### PR TITLE
Text for migration time validation.

### PR DESCRIPTION
**Pull request by molpopgen**

### CODE REVIEW

File: docs/specification.md [hunk 1]
```diff
@@ -914,6 +916,14 @@ Sorting in time-descending order avoids this discrepancy.
 Outline the basic logic of model validation
 :::
 
+#### Migration validation
+
+Migration start and end times are resolved in a pairwise fashion.
+(See {ref}`migration resolution <sec_spec_hdm_resolution_migration>`.)
+If the resolved `start_time` or the `end_time` of a migration is not
+contained by the time interval of BOTH `source` and `dest` demes,
```
Comment (jeromekelleher) [inline docs/specification.md:924]: I think the BOTH here will confuse people - the reason for the shouty MUST, SHOULD etc is to follow RFC 2119 as stated in the [conventions and terminoloty](https://popsim-consortium.github.io/demes-spec-docs/main/specification.html#conventions-and-terminology)

**Review by jeromekelleher — COMMENTED** [review]
LGTM. One nitpick re shouty words.

File: docs/specification.md (hunk 1 above)
Comment (molpopgen) [reply docs/specification.md]: Done.

**Review by grahamgower — APPROVED** [review]
LGTM, thanks @molpopgen.

**Pull request merged**